checkbox-ng/checkbox_ng/launcher/master.py

```diff
@@ -434,6 +436,8 @@ def register_arguments(self, parser):
         parser.add_argument(
             "-u", "--user", help=_("normal user to run non-root jobs")
         )
+        parser.add_argument("-n", "--dry-run", default=False, action="store_true", help=_(
```

Comment:
- this doesn't fit line length of PEP8
- `default=False` is redundant with `action="store_true"` and can be removed